var/spack/repos/builtin/packages/py-vcf-kit/package.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyVcfKit(PythonPackage):
+    """VCF-kit is a command-line based collection of utilities for performing
+       analysis on Variant Call Format (VCF) files."""
+
+    homepage = "https://github.com/AndersenLab/VCF-kit"
+    url      = "https://github.com/AndersenLab/VCF-kit/archive/0.1.6.tar.gz"
+
+    version('0.1.6', sha256='4865414ac9dc6996c0baeefadf1d528c28e6d0c3cc3dbdc28a2cdc6e06212428')
+
+    depends_on('python@2.7:2.8', type=('build', 'run'))
+    depends_on('py-setuptools', type='build')
+    depends_on('py-awesome-slugify', type=('build', 'run'))
+    depends_on('py-matplotlib', type=('build', 'run'))
+    depends_on('py-scipy', type=('build', 'run'))
+    depends_on('py-numpy', type=('build', 'run'))
+    depends_on('py-cython@0.24.1:', type='build')
+    depends_on('py-cyvcf2@0.6.5:', type=('build', 'run'))
+    depends_on('py-docopt', type=('build', 'run'))
+    depends_on('py-biopython', type=('build', 'run'))
+    depends_on('py-yahmm@1.1.2', type=('build', 'run'))
+    depends_on('py-clint', type=('build', 'run'))
```

Comment:
This dependency seems to be missing.

Reply:
`py-clint` was merged after this PR was opened, should now be rebased.